client/src/components/Recomendations/Recomendations.tsx

```
@@ -35,27 +37,45 @@ const Recomendations: React.FC = () => {
                 <ul className='flex items-center justify-center'>
                     <Marquee pauseOnHover vertical className="[--duration:20s]">
-                            {suggestions.map((suggestion, index) => (
-                                <div className="flex items-center border border-gray-300 rounded-lg">
-                                    <div className="p-4 flex items-center space-x-4">
-                                        <div className="h-6 w-6 rounded-full bg-gradient-to-r from-pink-500 to-violet-500 flex-shrink-0" />
-                                        <a href={`/user/${suggestion}`}>
-                                            <li key={index} className="list-none">@{suggestion}</li>
-                                        </a>
-                                        
+                        {suggestions.map((suggestion, index) => (
+                            <div className="flex items-center border border-gray-300 rounded-lg">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)